src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs

```diff
@@ -146,8 +146,41 @@ public override ExportResult Export(in Batch<Metric> batch)
                         }
                         else
                         {
-                            // TODO: Consider how/if to display buckets for exponential histograms.
-                            bucketsBuilder.AppendLine("Buckets are not displayed for exponential histograms.");
+                            var exponentialHistogramData = metricPoint.GetExponentialHistogramData();
+                            var scale = exponentialHistogramData.Scale;
+                            var factor = Math.Log(2) * Math.Pow(2, -scale);
+
+                            var offset = exponentialHistogramData.NegativeBuckets.Offset;
+                            List<string> negativeBucketCounts = null;
+                            foreach (var bucketCount in exponentialHistogramData.NegativeBuckets)
+                            {
+                                if (negativeBucketCounts == null)
+                                {
+                                    negativeBucketCounts = new List<string>();
+                                }
+
+                                var lowerBound = Math.Exp(offset * factor).ToString("F6", CultureInfo.InvariantCulture);
```

Comment:
Curious, where is this `F6` coming from, does it provide a good precision (rather than having two different buckets printed out the same thing)?